src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -1,6 +1,7 @@
 package uk.gov.nationalarchives.tdr.api.utils
 
 import java.sql.{PreparedStatement, Timestamp}
+import java.time.Instant
```

Comment:
Looks like there might be some unused imports to clear up. It's a shame ScalaStyle doesn't catch this!